api/enki.py

```diff
@@ -233,8 +228,6 @@ def reaper_request(self, identifier):
                 replace=ReplacementPolicy.from_license_source(self._db)
             )
 
-            self._db.commit()
```

Comment:
A monitor calls commit after every batch it processes (which is every 100 ids by default). 

(Leonard's out until Monday so I'm trying to answer his questions)